### PR TITLE
Make it a little less verbose

### LINKED ISSUE #416: kerl outputting result of uname | grep
I think this might be a recent addition to the code...

```
Extracting source code
Building Erlang/OTP 24.3.4.4 (24.3.4.4), please wait...
21.6.0
^ this
```

It confused me, since I was installing 24.3.4.4 and there was an extra output stating 21.6.0. It appears this output comes from https://github.com/kerl/kerl/blob/master/kerl#L858, which is potentially possible to hide. At this moment I just wanted to report this, so I don't forget it, and maybe in the future I might tackle it, if time allows and I don't forget 😄.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Closes #416.

@garazdawi, your review would be appreciated. 🙇

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Is there a way to check the CircleCI results without having to create an account for that service?

File: kerl [hunk 1]
```diff
@@ -782,8 +782,14 @@ maybe_patch_catalina() {
     fi
 }
 
+is_osx() {
+    uname="$(uname -r | grep "^$1.*$")"
+    [ "$uname" != "" ] && return
```
Comment (jadeallenx) [inline kerl:787]: I am not a huge fan of this construct - it's shorthand and cute but I'd prefer a more explicit and traditional if; then; else block, I guess.

Comment (paulo-ferraz-oliveira) [reply kerl:787]: Sure. No prob. 😄

Comment (paulo-ferraz-oliveira) [reply kerl]: How about 0f2ab1f? I may also change `return` to `return 0` and `false` to `return 1`, if you prefer.

File: kerl [hunk 2]
```diff
@@ -782,8 +782,16 @@ maybe_patch_catalina() {
     fi
 }
 
+is_osx() {
+    uname="$(uname -r | grep "^$1.*$")"
+    if [ "$uname" != "" ]; then
+        return
+    fi
+    false
```
Comment (garazdawi) [inline kerl:790]:
```suggestion
is_osx() {
    uname -r | \grep "^$1.*$" > /dev/null
```

I would have written it like this. It is the form used in many other places in the `kerl` code.

Comment (paulo-ferraz-oliveira) [reply kerl:790]: Cool. I can change that. It's more compact. Thanks. @jadeallenx, are you Ok with this suggestion?

Comment (jadeallenx) [reply kerl:790]: Yes, definitely prefer that, thanks

**Comment by jadeallenx** [comment]
> Is there a way to check the CircleCI results without having to create an account for that service?

I'm honestly pretty unhappy with CircleCI, no one maintains it and over time, and as usual it bit rots... I don't have a great solution for that.

**Comment by paulo-ferraz-oliveira** [comment]
> > Is there a way to check the CircleCI results without having to create an account for that service?
> 
> I'm honestly pretty unhappy with CircleCI, no one maintains it and over time, and as usual it bit rots... I don't have a great solution for that.

Would you be interested in moving to GitHub Actions (I have some experience with this - haven't looked into the CircleCI stuff on this repo., to be honest)? We could start by creating an issue and maybe then implement it later...

File: kerl (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply kerl]: Done. Thanks, @garazdawi, I just had to press the Commit suggestion button. 😄

**Comment by jadeallenx** [comment]
GHA is fine by me

**Comment by paulo-ferraz-oliveira** [comment]
btw, @jadeallenx, this is ready for review and merge. 😄

**Review by jadeallenx — APPROVED** [review]

**Comment by jadeallenx** [comment]
Thanks!!

**Pull request merged**